docs/publish.md

```
@@ -0,0 +1,43 @@
+There would also be a notification in the #bosh-private channel of the 
+Cloud Foundry slack.
+
+A USN triggers the stemcell build pipeline, when everything is successfully
+built, we can publish the new candidate stemcell version.
+Please check the latest build of the aggregate-candidate-stemcells job of the
+stemcell line you need to release (aggregate-candidate-stemcells-1.x),
+you should see a successful build which was triggered by the USN notice.
+Then, in the stemcells-publisher pipeline, check if google and aws light
+stemcells were successfully built as a result of the USN.
+
+## Publish stemcells
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)